lib/install/rust.js

```diff
@@ -0,0 +1,523 @@
+// System Objects
+var path = require('path');
+var spawn = require('child_process').spawn;
+var spawnSync = require('child_process').spawnSync;
```

Comment:
```js
var cp = require('child_process');
```

Then `cp.spawn` and `cp.spawnSync` throughout the file to be consistent throughout the project.